cli/Releasing.md

```diff
@@ -0,0 +1,129 @@
+# Release Checklist Template
+
+The Release Checklist Template can be generated with the following command: (replacing `X.XX.X` with the applicable release number):
+
+```
+go run main.go render checklist -v X.XX.X
+```
+
+The `--c` flag will copy the output to your clipboard. To view the output in the command line, simply remove the `--c` flag.
+
+
+If the release is a betafix/hotfix, include a message explaining briefly the reason for releasing the fix.
+**Example:** 
+
+```
+go run main.go render checklist -v X.XX.X --message "This hotfix addresses ISSUE_OR_CRASH in ORIGINAL_VERSION. See P2_PR_SLACK_ETC_LINK_OR_LINKS for the rationale behind the decision to make it."
+```
+
+# Different types of releases
+
+## Best practices
+
+It's best practice to use the automation script (mentioned in the release template above) for all releases types (alpha, regular, betafix, hotfix). When wrangling a betafix or hotfix, it's important to merge the fix to Gutenberg `trunk` independently of the release process. When the release is cut (by the automation script) the commit(s) that make up the betafix or hotfix should then be cherry-picked onto the Gutenberg release branch.
+
+## 1. Alpha
+
+### When
+
+Whenever a build is needed for testing (usually a few days prior to a Regular release).
+
+### Branches
+
+For example, when the next release will be `1.11.0`.
+
+| Repo             | Cut From | Branch Name                               |
+| ---------------- | -------- | ----------------------------------------- |
+| gutenberg        | trunk    | rnmobile/release_1.11.0-alpha1            |
+| gutenberg-mobile | trunk    | release/1.11.0-alpha1                     |
+| WPAndroid        | trunk    | gutenberg/integrate_release_1.11.0-alpha1 |
+| WPiOS            | trunk    | gutenberg/integrate_release_1.11.0-alpha1 |
+
+### Automation script differences
+
+Compared to a Regular release, the differences here are:
+
+- When the script asks for the new version number, don't forget to add the `-alpha` suffix (e.g. `1.11.0-alpha1`).
```

Comment:
@jhnstn Do we still do alpha releases? I'm not sure we need to keep this section.

Reply:
We still do. There are cases where we introduce dependent code via the native bridge in Gutenberg into the host apps. When that happens we create an alpha release so the editor does not break in the main apps. Doesn't happen often however.

I do think we need to formalize when an alpha release is needed. There might be other reasons besides the one I just mentioned.

Reply:
Also not sure how the CLI will handle the alpha version right now. I don't think it will work as-is.